mantis-control-plane/mantis-control-plane-server/src/test/java/io/mantisrx/master/resourcecluster/ResourceClusterActorClusterUsageAkkaTest.java

```diff
@@ -285,4 +285,6 @@ private static void assertIdleAndTotalCount(GetClusterUsageResponse usageRes, St
         assertEquals(idleCount, usage.getIdleCount());
         assertEquals(totalCount, usage.getTotalCount());
     }
+
+    // TODO: add test using sizeName....
```

Comment:
is this still valid?

Reply:
yeah, I just added a bit more tests on the getUsage path when TEs register with size name.